### PR TITLE
Added the ability to hold a local employee

### PR DESCRIPTION
**Pull request by Jh00291**
1) This will enable us to access the logged in employee across views to disable buttons and more.

### CODE REVIEW

File: code/java_project/src/edu/westga/cs3212/inventory_manager/Main.java [hunk 1]
```diff
@@ -65,6 +69,31 @@ public void start(Stage stage) throws IOException {
 	public static Stage getPrimaryStage() {
 		return primaryStage;
 	}
+	
+	/**
+	 * Sets the logged-in employee's credentials.
+	 * 
+	 * This method stores the credentials of the currently logged-in employee,
+	 * allowing other parts of the application to access user-related information.
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/Main.java:77]: Missing Pre and Post

File: code/java_project/src/edu/westga/cs3212/inventory_manager/Main.java [hunk 2]
```diff
@@ -65,6 +69,31 @@ public void start(Stage stage) throws IOException {
...
+	 * @param employee The credentials of the logged-in employee.
+	 */
+	public static void setLoggedInEmployee(LocalEmployeeCredentials employee) {
+		loggedInEmployee = employee;
+    }
+	
+	/**
+	 * Retrieves the logged-in employee's credentials.
+	 * 
+	 * This method returns the credentials of the currently logged-in employee,
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/Main.java:88]: Missing Pre and Post

**Review by JasonNunez — CHANGES_REQUESTED** [review]
Missing Pre and Post conditions for the methods in main

**Review by JasonNunez — APPROVED** [review]
LGTM!

**Review by jacobevans1 — APPROVED** [review]
Changes are correct.

**Pull request merged**